datamodels/2.x/itop-backup/dictionaries/hu.dict.itop-backup.php

```diff
@@ -22,39 +22,39 @@
  */
 Dict::Add('HU HU', 'Hungarian', 'Magyar', array(
 
-	'bkp-backup-running' => 'A backup is running. Please wait...~~',
-	'bkp-restore-running' => 'A restore is running. Please wait...~~',
+	'bkp-backup-running' => 'A backup fut. Kérem várjon...',
+	'bkp-restore-running' => 'A visszaállítás fut. Kérem várjon...',
 
-	'Menu:BackupStatus' => 'Scheduled Backups~~',
+	'Menu:BackupStatus' => 'Ütemezett Backup',
 	'bkp-status-title' => 'Scheduled Backups~~',
```

Comment:
@tacsaby can this one be translated to "Ütemezett Backup" as well? If so, I can apply the correction directly.

```suggestion
	'bkp-status-title' => 'Ütemezett Backup',
```

Reply:
I missed this...
'Ütemezett backup-ok'     is the correct sentence.
Thanks!

Reply:
So like this?
```suggestion
	'bkp-status-title' => 'Ütemezett backup-ok',
```

Reply:
Doesn't seem right, this is the title of the page, the "ok" should not be there right? It's not a status return message

Reply:
It seems like `-ok` is Hungarian for words in plural. But asking this confirmation from @tacsaby to be sure..

Reply:
My bad then 😅😬

Reply:
You can apply your last suggestion Thomas as it was the suggestion from tacsaby. Then I can move this PR forward.